Countries Info/GraphQL/Queries.graphql

```diff
@@ -0,0 +1,27 @@
+query MyCountryQuery {
```

Comment:
Usually queries are named based on subjects that they are related to, for example `CountryQuery`, this style `MyCountryQuery` is usually used for example purposes, lets switch to real project naming.